src/sentry/mail/notifications.py

```diff
@@ -99,12 +103,12 @@ def get_context(
     """
     context = {
         **shared_context,
-        **notification.get_user_context(user, extra_context),
+        **notification.get_recipient_context(recipient, extra_context),
     }
-    if notification.get_unsubscribe_key():
+    if isinstance(recipient, User) and notification.get_unsubscribe_key():
```

Comment:
Is this because we only build the unsubscribe link for emails and there is no team email? Can you leave a comment about this?

Reply:
The unsubscribe link is currently built with `user_id` so we can't look up the Team by ID. I'll leave a TODO.